notebooks/openvino/sentence_transformer_quantization.ipynb

```diff
@@ -0,0 +1,607 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Quantization of Text Embedding model from Sentence Transformers library"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%pip install optimum[openvino]\n",
+    "%pip install evaluate"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Quantize staticly model to 8-bit with NNCF via Optimum-Intel API"
```

Comment:
possibly adding some description about how to prepare dataset and how to define quantization config should be helpful.

Reply:
Done